apps/src/applab/applab.js

```diff
@@ -422,14 +416,23 @@ Applab.init = function (config) {
     }));
   }
   Applab.channelId = config.channel;
-  Applab.storage = initFirebaseStorage({
-    channelId: config.channel,
-    firebaseName: config.firebaseName,
-    firebaseAuthToken: config.firebaseAuthToken,
-    firebaseSharedAuthToken: config.firebaseSharedAuthToken,
-    firebaseChannelIdSuffix: config.firebaseChannelIdSuffix || '',
-    showRateLimitAlert: studioApp().showRateLimitAlert,
-  });
+
+  // TODO: post-firebase-cleanup, remove this conditional when we're removing firebase: #56994
+  if (!!config.useDatablockStorage) {
+    Applab.storage = initStorage(DATABLOCK_STORAGE, {
+      channelId: config.channel,
+    });
+  } else {
+    Applab.storage = initStorage(FIREBASE_STORAGE, {
+      channelId: config.channel,
+      firebaseName: config.firebaseName,
+      firebaseAuthToken: config.firebaseAuthToken,
+      firebaseSharedAuthToken: config.firebaseSharedAuthToken,
+      firebaseChannelIdSuffix: config.firebaseChannelIdSuffix || '',
+      showRateLimitAlert: studioApp().showRateLimitAlert,
```

Comment:
Speaking of rate limiting, are we doing any sort of rate limiting in the new datablock storage?

Reply:
Not yet, but that's next. https://github.com/code-dot-org/code-dot-org/issues/55481